### PR TITLE
Grouped sorceries fix / quest and gesture fixes

### PR DESCRIPTION
**Pull request by wdya**
Separated sorceries that were listed together in the same check box. Also:
- made some corrections to Seluvis's questline
- tweaked Miriel's questline
- corrected a couple gestures

### CODE REVIEW

File: data/sorceries.yaml [hunk 1]
```diff
@@ -6,8 +6,14 @@ sections:
     id: limgrave_sorceries
     num: 1
     items:
-      - [1, "Crystal Barrage/Glintstone Arc/Glintstone Pebble/Glintstone Stars/Scholar's Armament/Scholar's Shield - Sold by Sorceress Sellen under the Waypoint Ruins, East of the lake."]
-      - [2, "Carian Slicer/Glintblade Phalanx - Royal House Scroll located on the ruin next to a mage, East of the Agheel Lake South Grace in South-East Limgrave"]
+      - [1, "Crystal Barrage - Sold by Sorceress Sellen (under the Waypoint Ruins, East of the lake)."]
```
Comment (azy2) [inline data/sorceries.yaml:9]: Drop the 1 and 2 ids. If someone has checked this box we can't say for sure what they were thinking and we should just uncheck everything on the updated page.

File: data/sorceries.yaml [hunk 2]
```diff
@@ -60,7 +73,8 @@ sections:
     num: 6
     items:
       - [1, "Gravity Well - Just past the rotating elevator in Raya Lucaria, next to the Schoolhouse Grace. So the quick/tricky method is from the Grace, time jumping down to the rising elevator then hop down to the path below. If you can't do this, then come from the Church of the Cuckoo Grace and run past all the enemies. Past the elevator is a bunch of graves where an enemy will spawn. Kill it for the spell."]
-      - [2, "Glintstone Cometshart/Star Shower - Conspectus Scroll. Go to the room past the Schoolhouse Classroom Grace and turn left to find a room with the scroll inside."]
+      - [2, "Glintstone Cometshard - Unlocked by Conspectus Scroll. Go to the room past the Schoolhouse Classroom Grace and turn left to find a room with the scroll inside."]
```
Comment (azy2) [inline data/sorceries.yaml:76]: same here

File: data/sorceries.yaml [hunk 3]
```diff
@@ -81,7 +95,8 @@ sections:
       - [3, "Night Comet - One of the chests in Sellia that are available after unlocking the seals by lighting the towers."]
       - [4, "Crystal Torrent - Kill the Boss of Sellia Hideaway. To find the hideaway, go to Sellia in the North-East part of the swamp. Go past the seals by lighting the fires at the tops of the towers. Go up the path to the Church of Plague. Go North of the Church to find the mage enemy and a giant gravestone. Roll into the cliff behind the Grave to break the Illusion and find the Hideaway."]
       - [5, "Eternal Darkness - Go South from the Church of the Plague(explained just above) and cross the top of the gate nearby then go into the Swamp Lookout Tower to find the spell."]
-      - [6, "Night Maiden's Mist/Night Shard - Bought from Gowry South of Sellia after finishing his request."]
+      - [6, "Night Maiden's Mist - Sold by Gowry in Gowry's Shack south of Sellia after finishing his request."]
```
Comment (azy2) [inline data/sorceries.yaml:98]: and here

File: data/sorceries.yaml [hunk 4]
```diff
@@ -100,7 +115,8 @@ sections:
     id: altus_plateau_sorceries
     num: 10
     items:
-      - [1, "Unseen Blade/Unseen Form - At the tower Mirage Rise located in North Altus Plateau, at the end of the path going West from the Minor Erdtree. To solve the puzzle, touch the three seals. One is South-West, One is East, and one is further East hidden by an Illusory Rock."]
```
Comment (azy2) [inline data/sorceries.yaml:103]: and finally here

**Review by azy2 — CHANGES_REQUESTED** [review]

**Comment by wdya** [comment]
How's that?

**Comment by azy2** [comment]
Looks great! Thank you.

**Pull request merged**